pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -59,6 +60,9 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 			newQuery = strings.Replace(r.RequestURI, r.URL.Query().Get("name"), r.URL.Query().Get("name")+r.Header.Get(headers.AuthZTenantIdHeaderName), 1)
 			config.Config.Labels[headers.OriginalNameLabel] = r.URL.Query().Get("name")
 		}
+		// Replace network reference with network ID, force Network-scoped alias for DNS use.
```

Comment:
Use case: user supplied --net and --net-alias without --name. We didn't support this use case until now.